lldb/test/API/lang/swift/async/tasks/info/TestSwiftTaskInfo.py

```diff
@@ -0,0 +1,25 @@
+import lldb
+from lldbsuite.test.decorators import *
+from lldbsuite.test.lldbtest import TestBase
+import lldbsuite.test.lldbutil as lldbutil
+
+import re
+
+
+def _tail(output):
+    """Delete the first line of output text."""
+    result, _ = re.subn(r"^.*\n", "", output, count=1)
+    return result
+
+
+class TestCase(TestBase):
+    def test(self):
+        self.build()
+        lldbutil.run_to_source_breakpoint(
+            self, "break here", lldb.SBFileSpec("main.swift")
+        )
+        self.runCmd("frame variable task")
```

Comment:
This is an unusual way of checking the output. I'd have either expected self.expect or using fame.FindVariable()?

Reply:
It is unusual. While I could use `FindVariable` to for one half of this test (`frame variable`), the other half of this test, `task info`, is all text. Since `task info` is text, then it seems the the most straightforward thing to do is get the output of `frame variable` as text too.

Reply:
Ah I see, you can't do the assertEqual if you were using expect!